finagle-core/src/main/scala/com/twitter/finagle/service/StatsFilter.scala

```diff
@@ -1,11 +1,14 @@
 package com.twitter.finagle.service
 
+import java.util.concurrent.atomic.LongAdder
+import java.util.concurrent.TimeUnit
```

Comment:
sorry, do you mind sorting these with everything else?  as in, com comes before java, java comes before scala, etc.